commec/screen-default-config.yaml

```
@@ -23,6 +23,9 @@ databases:
     annotations: '{default}low_concern/low_concern_annotations.tsv'
   taxonomy:
       path: "{default}taxonomy/"
+  regulated_lists:
+      path: "{default}regulated_lists"
```

Should we use `control_lists` instead of `regulated_lists`?
```suggestion
  control_lists:
      path: "{default}control_lists"
```